assets/src/edit-story/components/canvas/frameElement.js

```diff
@@ -87,7 +88,9 @@ function FrameElement({ element }) {
         if (!isSelected) {
           handleSelectElement(id, evt);
         }
-        evt.stopPropagation();
+        if (!isBackground) {
```

Comment:
Why is this? Could you add a comment in code. This seems like an important behavior involving background elements. As such, I'm also wondering if it'd be safer to roll it all inside the `handleSelectElement`? Otherwise we're fragmenting the behavior a bit between two places. It used to be easier with a simple `stopPropagation`, but this could be more nuanced.